src/Swup/assets/src/index.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module 'swup';
+declare module '@swup/debug-plugin';
+declare module '@swup/forms-plugin';
+declare module '@swup/fade-theme';
+declare module '@swup/slide-theme';
```

Comment:
Can you tell more about this? We are not declaring any .d.ts files by hand at the moment. Are you fixing missing types from those swup libs?

Reply:
Fairly speaking, I don't know for sure, but that's something that the warning suggested to do:
```
(!) Plugin typescript: @rollup/plugin-typescript TS7016: Could not find a declaration file for module 'swup'. '/Users/victor/www/symfony/ux/node_modules/swup/lib/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/swup` if it exists or add a new declaration (.d.ts) file containing `declare module 'swup';`
```

And since no `@types/swup` package, I did the 2nd suggestion. I just googled how to do it and it worked in this case